events/ready.js

```diff
@@ -24,20 +24,24 @@ module.exports = async (Discord, client) => {
 
 	let presences = [
 		["PLAYING", `See the latest updates by using "@${client.user.username} changelog"`],
-		["WATCHING", `${(await Suggestion.countDocuments())} suggestions`],
+		["WATCHING", async () => `${(await Suggestion.countDocuments())} suggestions`],
 		["PLAYING", `Vote for Suggester and get rewards! Use "@${client.user.username} vote" for more info`],
 		["PLAYING", `Join our support server! Use "@${client.user.username}" support for more info`]
 	];
 
 	let p = 0;
-	function setPresence() {
+	async function setPresence() {
 		let presence = presences[p];
-		client.user.setActivity(`${presence[1]} • @${client.user.username} help`, { type: presence[0] });
+		let type = presence[0];
+		let text = presence[1];
+		if (typeof text == "function")
+			text = await text();
+		client.user.setActivity(`${text} • @${client.user.username} help`, { type });
 		p = p+1 === presences.length ? 0 : p+1;
 	}
-	setPresence();
-	setInterval(function() {
-		setPresence();
+	await setPresence();
+	setInterval(async function() {
+		await setPresence();
```

Comment:
I mean, it's a single line of code, it doesn't really do much changing it to async/await

Reply:
Ok, i understand, but why is it with async/await in https://github.com/Suggester-Bot/Suggester/pull/167/files#diff-de0c46c47a38d81005e2f35b90f8d9dbR56 ?